src/libraries/System.Runtime.InteropServices.Experimental/System.Runtime.InteropServices.Experimental.sln

```diff
@@ -0,0 +1,198 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
@safern, @ericstj could you give this commit (https://github.com/dotnet/runtime/pull/64556/commits/7ae9004393332c2274628f09d4cf787a276485b5) a look and make sure it looks correct?

I think the biggest difference between S.Runtime and S.R.InteropServices is that the former doesn't actually include a src project while the latter does and I'm not 100% if matching S.Runtime is "correct" here (based on what I know about NuGet, I'd guess no and I need to also include a "copy" of the source assembly with the same name/contents).

---

Noting that we still haven't finalized on the naming here. We are likely to change this away from `System.Runtime.InteropServices.Experimental` and possibly to something else as this isn't truly an "experimental" package; but rather something for Xamarin/MAUI to get access to operators on NFloat for the LTS.

Reply:
Looks good to me.